src/contract/deploy.rs

```diff
@@ -133,12 +146,12 @@ mod tests {
             let builder = Contract::deploy(api::Eth::new(&transport), include_bytes!("./res/token.json")).unwrap();
 
             // when
-            builder.options(Options::with(|opt| opt.value = Some(5.into()))).confirmations(1).execute(vec![1, 2, 3, 4], (U256::from(1_000_000), "My Token".to_owned(), 3u64, "MT".to_owned()), 5.into()).unwrap().wait().unwrap();
+            builder.options(Options::with(|opt| opt.value = Some(5.into()))).confirmations(1).execute("0x01020304", (U256::from(1_000_000), "My Token".to_owned(), 3u64, "MT".to_owned()), 5.into()).unwrap().wait().unwrap();
         };
 
         // then
         transport.assert_request("eth_sendTransaction", &[
-      "{\"data\":\"0x0102030400000000000000000000000000000000000000000000000000000000000f42400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000084d7920546f6b656e00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024d54000000000000000000000000000000000000000000000000000000000000\",\"from\":\"0x0000000000000000000000000000000000000005\",\"value\":\"0x5\"}".into(),
+      "{\"data\":\"0x303130323033303400000000000000000000000000000000000000000000000000000000000f42400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000084d7920546f6b656e00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024d54000000000000000000000000000000000000000000000000000000000000\",\"from\":\"0x0000000000000000000000000000000000000005\",\"value\":\"0x5\"}".into(),
```

Comment:
This test should not be changed - currently the change is required due to incorrect `into_bytes()` conversion.

Reply:
Thanks!